src/System.Reflection/tests/AssemblyNameTests.cs

```diff
@@ -302,10 +302,9 @@ public void FullName(string name, string expectedName)
             AssemblyName assemblyName = new AssemblyName(name);
 
             expectedName = expectedName.ToLowerInvariant();
```

Comment:
Nit: Are the `ToLowerInvariant` calls needed - remove them too? I would not expect that assembly name parsing and formatting flips the case.

Reply:
I thought that too initially, but they were added in https://github.com/dotnet/corefx/commit/617a9e944c3946ced0c2a671ce9ec08744de9a4b so I kept them around.

Reply:
That change was compensating for .NET Framework bug. We do not run these tests on .NET Framework anymore. It should be fine to remove these.

Reply:
Ah okay, will do.